File: Easy/JAVA/TotalSpeedBicycle.java
import java.util.Arrays;
import java.util.Collections;

/**
 * TotalSpeedBicycle
 */
public class TotalSpeedBicycle {

    private static int TandemBicycle(int[] redTeam, int[] blueTeam, boolean fastest) {
        Arrays.sort(redTeam);
        Arrays.sort(blueTeam);

        if (fastest)
            Collections.reverse(Arrays.asList(blueTeam));

        int totalMaxSpeed = 0;
        for (int i = 0; i < blueTeam.length; i++)
            totalMaxSpeed += Math.max(redTeam[i], blueTeam[i]);

        return totalMaxSpeed;
    }

    public static void main(String[] args) {
        int[] redTeam = new int[] { 3, 4, 4, 1, 1, 8, 9 };
        int[] blueTeam = new int[] { 9, 8, 2, 2, 3, 5, 6 };
        boolean fastest = false;
        int result = TotalSpeedBicycle.TandemBicycle(redTeam, blueTeam, fastest);
        System.out.println(result);
    }
}

File: Easy/JAVASCRIPT/TotalSpeedBicycle.js
const totalSpeedBicycle = (redTeam, blueTeam, fastest) => {
    redTeam.sort((a, b) => (a - b));
    blueTeam.sort((a, b) => (a - b));

    if (fastest)
        blueTeam.reverse();
    
    let totalMaxSpeed = 0;
    for (let i = 0; i < blueTeam.length; i++)
        totalMaxSpeed += Math.max(redTeam[i], blueTeam[i]);

    return totalMaxSpeed;
}

const redTeam = [3, 4, 4, 1, 1, 8, 9];
const blueTeam = [9, 8, 2, 2, 3, 5, 6];
const fastest = true;
const result = totalSpeedBicycle(redTeam, blueTeam, fastest);
console.log(result);
